docs/StardustDocs/topics/info/tail.md

```diff
@@ -0,0 +1,55 @@
+# tail
+
+
+<web-summary>
+Discover `tail` operation in Kotlin Dataframe.
+</web-summary>
+
+<card-summary>
+Discover `tail` operation in Kotlin Dataframe.
+</card-summary>
+
+<link-summary>
+Discover `tail` operation in Kotlin Dataframe.
+</link-summary>
+
+<!---IMPORT org.jetbrains.kotlinx.dataframe.samples.api.info.TailSamples-->
+
+Returns a [`DataFrame`](DataFrame.md) with the last `numRows` rows.
+
+This is equivalent to calling [`takeLast`](sliceRows.md#takelast) with the same argument. By default, `numRows = 5`.
+
```

Comment:
where's head()? tail and head form a pair. They should link to each other or share descriptions.